Workbooks/KeyVault/Overview/settings.json

```diff
@@ -2,5 +2,5 @@
     "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
     "name":"Key Vault Overview",
     "author": "Microsoft",    
-    "galleries": [{ "type": "keyvaults-insights", "resourceType": "microsoft.keyvault/vaults", "order": 100 }]
+    "galleries": [{ "type": "keyvaults-insights", "resourceType": "Microsoft.KeyVault/vaults", "order": 100 }]
```

Comment:
should this be capitalized? I noticed in the current keyvault resource blade, when you click on the UsageNotebookBlade in debugger mode on mpac (ctrl+alt+d), in the console you can find that GalleryResourceType is undefined, 
![image](https://user-images.githubusercontent.com/7664931/81222593-5d853280-8f99-11ea-8862-274f0b1ef873.png)
so no templates will show. I think we will need to configure the menu item in keyvaults so we can assign a gallery type, like we do with storage accounts.

![image](https://user-images.githubusercontent.com/7664931/81222650-77bf1080-8f99-11ea-835a-bda60a40073d.png)

Reply:
if the gallery resource type isn't *explicitly* set, it comes from the resource type of componentId.  in the above case it would still be `microsoft.keyvault/vaults`
i'm not sure why "source" in your bottom screenshot is also a resource id, it shouldn't be.  it should be the source of the blade, not a resource id.  it's only used for telemetry, but having it be a resource means it has a huge cardinality.

Reply:
I see, this was something I did for storage accounts, I'll submit a pr to fix the source